### PR TITLE
chore(docs): improved mission descriptions

### LINKED ISSUE #217: RHOAR Launch Experience - Update mission & runtime descriptions
The descriptions for missions and runtimes in the RHOAR (and OSIO) launcher are out-of date. Please update them according to the [launch content document, page 5-6](https://docs.google.com/document/d/1PRKPlg9zOgG9Eu5GZr96GCADPhPnLwZte0jkVx5q3KQ/edit?usp=sharing). Stacy worked with stakeholders on finalizing the text for them.

cc @gastaldi

### PR DESCRIPTION
**Pull request by gastaldi**
Fixes #217

### CODE REVIEW

File: metadata.yaml [hunk 1]
```diff
@@ -1,27 +1,27 @@
 missions:
 - id: cache
   name: Cache
-  description: The Cache mission demonstrates how to use a cache to improve the response time of applications.
+  description: The Cache mission demonstrates how to use a cache to improve the response time of applications
```
Comment (rhoads-zach) [inline metadata.yaml]: Going with your theme, this should be something like:

> Use a cache to improve the response time of applications

File: metadata.yaml [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
 - id: crud
   name: CRUD
-  description: The Relational Database Backend mission expands on the REST API Level 0 mission to provide a basic example of performing create, read, update and delete (CRUD) operations on a PostgreSQL database using a simple HTTP API. CRUD operations are the four basic functions of persistent storage, widely used when developing an HTTP API dealing with a database.
+  description: Perform operations on a PostgreSQL database using a simple HTTP API
   metadata:
     level: foundational
 - id: health-check
   name: Health Check
-  description: The Health Check mission demonstrates the health check pattern. When you deploy an application, its important to know if it is available and if it can start handling incoming requests. Implementing the health check pattern allows you to monitor the health of an application, which includes if an application is available and whether it is able to service requests.
+  description: Monitor the health of an application, including an application’s availability and its ability to service requests
```
Comment (rhoads-zach) [inline metadata.yaml]: I'm confused by this:
> including an application’s availability and its ability to service requests

Are you trying to say something like:
>  including the functionality of an application and its ability to service requests

File: metadata.yaml [hunk 3]
```diff
@@ -50,12 +50,12 @@ missions:
...
   name: REST API Level 0
-  description: The REST API Level 0 mission shows how to map business operations to a remote procedure call endpoint over HTTP using a REST framework. This corresponds to Level 0 in the Richardson Maturity Model. Creating an HTTP endpoint using REST and its underlying principles to define your API lets you quickly prototype and design the API flexibly.
+  description: Map business operations to a remote procedure call endpoint over HTTP using a REST framework
   metadata:
     level: foundational
 - id: rest-http-secured
   name: Secured
-  description: The Secured mission secures a REST endpoint using Red Hat SSO and expands on the REST API Level 0 mission. It implements the Open ID Connect protocol which is an extension of the OAuth 2.0 specification and issues access tokens to provide clients with various access rights to secured resources. Securing an application with Red Hat SSO enables you to add security to your applications while centralizing the security configuration.
+  description: Expands on REST API Level 0 - secure a REST endpoint using Red Hat SSO to add security to your applications while centralizing security configuration
```
Comment (rhoads-zach) [inline metadata.yaml]: Suggest to reword as:
> Expands on REST API Level 0 to secure the REST endpoint using Red Hat SSO which adds security to your applications while centralizing the security configuration

File: metadata.yaml (hunk 2 above)
Comment (gastaldi) [reply metadata.yaml]: I'm just copying what's in https://docs.google.com/document/d/1PRKPlg9zOgG9Eu5GZr96GCADPhPnLwZte0jkVx5q3KQ/edit# :wink:

Comment (stacymcauliffe) [reply metadata.yaml]: This change looks good to me, thanks @rhoads-zach @gastaldi !

File: metadata.yaml [hunk 4]
```diff
@@ -1,27 +1,27 @@
...
 - id: configmap
   name: Externalized Configuration
-  description: The Externalized Configuration mission provides a basic example of using a ConfigMap to externalize configuration. ConfigMap is an object used by OpenShift to inject configuration data as simple key and value pairs into one or more Linux containers while keeping the containers independent of OpenShift.
+  description: Use a ConfigMap to externalize configuration. ConfigMap is an object used by OpenShift to inject configuration data as simple key and value pairs into one or more Linux containers while keeping the containers independent of OpenShift
   metadata:
     level: foundational
 - id: crud
   name: CRUD
-  description: The Relational Database Backend mission expands on the REST API Level 0 mission to provide a basic example of performing create, read, update and delete (CRUD) operations on a PostgreSQL database using a simple HTTP API. CRUD operations are the four basic functions of persistent storage, widely used when developing an HTTP API dealing with a database.
+  description: Perform operations on a PostgreSQL database using a simple HTTP API
```
[the remark this answers is not in the document]
Comment (kenfinnigan) [reply metadata.yaml]: @stacymcauliffe I'd probably lean towards the latter statement, but that's a personal preference

[the remark this answers is not in the document]
Comment (stacymcauliffe) [reply metadata.yaml]: ok! I'll update the doc. Thanks for the help

[the remark this answers is not in the document]
Comment (gastaldi) [reply metadata.yaml]: image

[the remark this answers is not in the document]
Comment (gastaldi) [reply metadata.yaml]: @stacymcauliffe I'm fine with that too :wink: Let me update this PR

[the remark this answers is not in the document]
Comment (stacymcauliffe) [reply metadata.yaml]: hahahah thank you!

**Comment by gastaldi** [comment]
@stacymcauliffe WDYT of having the Health Check description as `Monitor the ability of an application to service requests`?

**Comment by stacymcauliffe** [comment]
@gastaldi, that works. "Monitor an application's ability to service requests" is a bit simpler, but I'm fine with either.

**Comment by gastaldi** [comment]
I like your suggestion, let's use that instead

**Pull request merged**